File: src/DataStructure/Array/1D/PivotIndex/python/testCases.py
import json

try:
    data = open('testCases.json')
except FileNotFoundError:
    from pathlib import Path
    data = (Path(__file__).parent / '../testCases.json').open()

cases = json.load(data)


def runTestCase(findPivot, testCase):
    return findPivot(testCase['input'])

File: src/DataStructure/HashMap/Easy/MyHashMap/javascript/testCases.js
import json from '../testCases.json';

export const cases = json;

export default (HashMap, { input, args }) => {
  const map = new HashMap();
  return input.map((command, i) => {
    const func = HashMap.prototype[command];
    if (func) return func.apply(map, args[i]);
    return null;
  });
};
